### ISSUE TITLE
Issue with face bounding box coordinates in multi-person video – audio only, no visuals

### ISSUE THREAD

**Issue by Hissatsu265** [MeiGen-AI/MultiTalk#40]
https://github.com/user-attachments/assets/2d899865-6fa2-4958-a0c7-2f6cc829eb75

Hi, thanks for the great model!

I'm trying to generate a multi-person talking video by specifying the bounding box coordinates for both faces. I set the origin at the bottom-right corner of the image, with the X-axis going upwards and the Y-axis going from right to left.

However, the generated output only contains audio — there's no video/image shown at all.

My settings:

steps: 12

Input: bounding boxes of both faces with the coordinate system as described above.

Could you please clarify what coordinate system the model expects for multi-person setups? Am I doing something wrong with the direction or origin? this is link to my resource https://drive.google.com/drive/folders/1mHl8chhCkFE0nWXSCUJ-TWONb3Me_2c4?usp=sharing

Thanks in advance!

**Comment by kongzhecn** [comment]
I found that there are some errors in the bbox you provided. You can refer to the following code to visualize your bbox.
```
import torch
import numpy as np
from PIL import Image
import matplotlib.pyplot as plt
import json

def visualize_mask(input_data):
    img = Image.open(input_data['cond_image']).convert("RGB")
    src_w, src_h = img.size  
    img_np = np.array(img)
    background_mask = torch.zeros([src_h, src_w])

    human_masks = []
    for _, person_bbox in input_data['bbox'].items():
        x_min, y_min, x_max, y_max = person_bbox
        human_mask = torch.zeros([src_h, src_w])
        human_mask[int(x_min):int(x_max), int(y_min):int(y_max)] = 1
        background_mask += human_mask
        human_masks.append(human_mask)
    background_mask = torch.where(background_mask > 0, torch.tensor(0), torch.tensor(1))
    human_masks.append(background_mask)

    plt.figure(figsize=(15, 10))
    
    plt.subplot(2, 3, 1)
    plt.imshow(img_np)
    plt.title("Original Image")
    plt.axis('off')
    
    plt.subplot(2, 3, 2)
    plt.imshow(human_masks[0].numpy(), cmap='gray')
    plt.title("Foreground Mask 1")
    plt.axis('off')
    
    plt.subplot(2, 3, 3)
    plt.imshow(img_np)
    plt.imshow(human_masks[0].numpy(), cmap='Reds', alpha=0.4)
    plt.title("Image + Foreground 1")
    plt.axis('off')
    
    plt.subplot(2, 3, 4)
    plt.imshow(human_masks[1].numpy(), cmap='gray')
    plt.title("Foreground Mask 2")
    plt.axis('off')
    
    plt.subplot(2, 3, 5)
    plt.imshow(img_np)
    plt.imshow(human_masks[1].numpy(), cmap='Greens', alpha=0.4)
    plt.title("Image + Foreground 2")
    plt.axis('off')
    
    plt.subplot(2, 3, 6)
    plt.imshow(human_masks[2].numpy(), cmap='gray')
    plt.title("Background Mask")
    plt.axis('off')
    
    plt.tight_layout()
    plt.savefig('mask_visualization.png', bbox_inches='tight')
    plt.show()

    

if __name__ == '__main__':
    input_json_path = 'examples/multitalk_example_1.json'
    with open(input_json_path, 'r', encoding='utf-8') as f:
        input_data = json.load(f)
    visualize_mask(input_data)

```
The following are the visualization results of `examples/multitalk_example_1.json` and the file you provided, respectively.

`examples/multitalk_example_1.json`:
![Image](https://github.com/user-attachments/assets/a99e7f04-2333-4c69-9fc4-b186c6b6367a)

Your json file:
![Image](https://github.com/user-attachments/assets/b64b466e-c05b-4bcd-86d0-535ea6d4daae)
